templates/index-public.php

```diff
@@ -90,9 +90,9 @@
 		<div id="screens"></div>
 
 		<div id="emptycontent">
-			<div id="emptycontent-icon" class="icon-video"></div>
+			<div id="emptycontent-icon" class="icon-talk"></div>
 			<h2><?php p($l->t('Join a conversation or start a new one')) ?></h2>
-			<p class="uploadmessage"></p>
+			<p class="emptycontent-additional">👋 Say hi to your friends and colleagues!</p>
```

Comment:
This is not translated

Reply:
Please avoid the emoji until we ship an emoji font in nextcloud

Reply:
@nickvergessen removed, and fixed translatability. Thanks! :)